operate/qa/integration-tests/src/test/java/io/camunda/operate/util/OperateTester.java

```diff
@@ -966,4 +978,11 @@ public OperateTester withAuthenticationToken(final String token) {
     SecurityContextHolder.getContext().setAuthentication(jwtAuthenticationConverter.convert(jwt));
     return this;
   }
+
+  public void performOneRoundOfImport() {
+    zeebeImporter.performOneRoundOfImport();
```

Comment:
💭 ❓ We might need to move this into the `.untilAsserted` so the trigger is repeated? 🤔 What do you think? Can this be missed?

Reply:
❓ Is it sufficient to simply check for the active job count in the executor 🤔 

Does it maybe make sense to check the import position? First get position - X. Then schedule and wait until it is larger then X?

Reply:
Looking at the code it might be sufficient